packages/ibm-products-web-components/src/components/about-modal/about-modal.test.ts

```diff
@@ -75,9 +79,144 @@ const template = (props = defaultProps) => html`
+    const modalBody = element.shadowRoot?.querySelector(
+      `${carbonPrefix}-modal-body`
+    );
+    expect(modalBody).toBeTruthy();
+    expect(modalBody?.shadowRoot).toBeTruthy();
+    const slot = modalBody?.shadowRoot?.querySelector('slot');
+    expect(slot).toBeTruthy();
+    const assignedElements = slot?.assignedElements({ flatten: true }) ?? [];
+    const modalCopyRight = assignedElements
+      .map((el) => el.querySelector(`.${blockClass}__copyright-text`))
+      .find((el) => el !== null);
+    expect(modalCopyRight).toBeTruthy();
+    expect(modalCopyRight?.textContent?.trim()).toBe(
+      defaultProps.copyrightText
+    );
+  });
+  it('renders links', async () => {
+    const element: CDSAboutModal = await fixture(template());
+    const modalBody = element.shadowRoot?.querySelector(
+      `${carbonPrefix}-modal-body`
+    );
+    expect(modalBody).toBeTruthy();
+    expect(modalBody?.shadowRoot).toBeTruthy();
+    const slot = modalBody?.shadowRoot?.querySelector('slot');
+    expect(slot).toBeTruthy();
+    const assignedElements = slot?.assignedElements({ flatten: true }) ?? [];
+    if (element.links.length > 0) {
+      const modalLinkContainer = assignedElements
+        .map((el) => el.querySelector(`.${blockClass}__links-container`))
+        .find((el) => el !== null);
+      expect(modalLinkContainer).toBeTruthy();
+      const links = modalLinkContainer?.querySelectorAll(
+        `${carbonPrefix}-link`
+      );
+      expect(links?.length == element?.links.length).toBeTruthy();
+    }
+  });
+  it('renders additional info', async () => {
+    const element: CDSAboutModal = await fixture(template());
+    if (element.additionalInfo) {
+      const modalFooter = element.shadowRoot?.querySelector(
+        `${carbonPrefix}-modal-footer`
+      );
+      expect(modalFooter).toBeTruthy();
+    }
+  });
+  it('closes modal on clicking close button', async () => {
+    const element: CDSAboutModal = await fixture(template());
+    const modalClose = element.shadowRoot?.querySelector(
+      `${carbonPrefix}-modal-close-button`
+    );
+    expect(modalClose).toBeTruthy();
+    expect(element.open).toBe(true);
+    (modalClose as HTMLElement)?.click();
+    expect(element.open).toBe(false);
+  });
+  it('Scrollable body with overflowing content', async () => {
+    const element: CDSAboutModal = await fixture(template());
+    const container = element.shadowRoot?.querySelector(
+      `${carbonPrefix}-modal-body`
+    ) as HTMLElement;
+    // Simulate overflow: content fits
+    Object.defineProperty(container, 'scrollHeight', {
+      value: 200,
+      configurable: true,
+    });
+    Object.defineProperty(container, 'clientHeight', {
+      value: 100,
+      configurable: true,
+    });
+    // Manually call _checkOverflow to force the overflow check
+    element['_checkOverflow']();
+    await element.updateComplete;
+    expect(
+      container.classList.contains('c4p--about-modal-scroll-content')
```

Comment:
There's a few places in this test where I see `c4p` hardcoded in classes. We should pull in the `prefix` from `globals/settings.ts`.